tests/client-sdk/agents/test_agents.py

```diff
@@ -263,7 +263,8 @@ def test_custom_tool(llama_stack_client, agent_config):
     assert "CustomTool" in logs_str
 
 
-def test_override_system_message_behavior(llama_stack_client, agent_config):
+# TODO: fix this flaky test
```

Comment:
Might be good to add a link to issue to track

Reply:
adding a `@pytest.skip("Skipping because flakeyyyyyy")` might also be a good idea so it stares at us in the logs :)